Workbooks/AsdEssential8.json

```diff
@@ -0,0 +1,3482 @@
+                    "size": 4,
+                    "timeContextFromParameter": "TimeRange",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "crossComponentResources": [
+                      "{Workspace}"
+                    ],
+                    "visualization": "piechart",
+                    "chartSettings": {
+                      "seriesLabelSettings": [
+                        {
+                          "seriesName": "Unhealthy",
+                          "color": "red"
+                        },
+                        {
+                          "seriesName": "Healthy",
+                          "color": "green"
+                        }
+                      ]
+                    }
+                  },
+                  "customWidth": "50",
+                  "name": "query - 19"
+                }
+              ]
+            },
+            "conditionalVisibility": {
+              "parameterName": "Tab",
+              "comparison": "isEqualTo",
+              "value": "General"
+            },
+            "name": "group - 12"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "General"
+      },
+      "name": "group - 14",
+      "styleSettings": {
+        "margin": "25px"
+      }
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
Hey @nickt444, A workbook template must not contain any references to resources. Please keep an empty array, thanks.

Reply:
done